waspc/src/Wasp/Generator/Crud/Routes.hs

```diff
@@ -0,0 +1,21 @@
+module Wasp.Generator.Crud.Routes where
+
+import Data.List (intercalate)
+import qualified Wasp.AppSpec.Crud as AS.Crud
+
+getRoute :: AS.Crud.CrudOperation -> String
+getRoute operation = case operation of
+  AS.Crud.Get -> "get"
+  AS.Crud.GetAll -> "getAll"
+  AS.Crud.Create -> "create"
+  AS.Crud.Update -> "update"
+  AS.Crud.Delete -> "delete"
+
+makeFullPath :: String -> AS.Crud.CrudOperation -> String
+makeFullPath crudOperationName crudOperation = intercalate "/" [crudOperationsRouterNamespace, getCrudOperationRouterRoute crudOperationName, getRoute crudOperation]
+
+crudOperationsRouterNamespace :: String
+crudOperationsRouterNamespace = "crud"
```

Comment:
Implementation difficulties aside, what do you think about not having the string `crud` as a part of the route?

I think it's cleaner. In fact, I'd prefer hiding the word "crud" from our users in as many places as we can (including the naming our documentation suggests for their stuff). It kinda feels too low level for me.

Perhaps we can even reconsider naming the declaration `crud` and the feature `Auto CRUD` and replace them with something that's more easily marketable.

Meteor uses the word "collection" for a similar feature. I can't remember what Rails uses, but it's worth checking out.

Reply:
I like the thinking here!
Although, I am not sure what to call it hm. "resource"? Somehow CRUD feels very relevant.
"collection" is coming from mongo I believe, they have "documents" and "collections", and meteor was based heavily on mongo.
"aggregate" :D?
I don't know, I keep coming back to crud. I am open to something better, but it is hard to think of something better.

Exercise: imagine naming it "resource" or "collection". Somebody reads that, asks what is it? What do you tell them? You tell them "it is CRUD" :D.